### PR TITLE
Added pip install ninja to accelerate installation of flash-attn

### LINKED ISSUE #1460: Added installation of ninja to Quickstart
### What piece of documentation is affected?

The Quickstart section in README.md 

### What part(s) of the article would you like to see updated?

the Quickstart section of the README.md

pip3 install ninja is required before 
pip3 install -e '.[flash-attn,deepspeed]'

ninja helps using all the cores, else it takes more than 2hrs and defeates the "Quickstart" section

### Additional Information

_No response_

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this feature has not been requested yet.
- [X] I have provided enough information for the maintainers to understand and evaluate this request.

### PR DESCRIPTION
**Pull request by melvinebenezer**
# Description
In the Quickstart section, it is required to install flash-attn
Without ninja , installing flash-attn takes more than 2hrs on 64cores

## Motivation and Context

The Quickstart section is expected to get started quickly. however the installation of flash-attn takes a long time
fixes #1460 

## How has this been tested?
Only README.md has changed

## Screenshots (if appropriate)
Screenshot 2024-03-30 at 11 46 47 AM

## Types of changes

NA

## Social Handles (Optional)
twitter: @jamesmel24
discord: jamesmel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1305,6 +1306,7 @@ git clone https://github.com/OpenAccess-AI-Collective/axolotl
 cd axolotl
 
 pip3 install packaging
+pip3 install ninja
```
Comment (NanoCode012) [inline README.md]:
```suggestion
pip3 install packaging ninja
```

File: README.md [hunk 2]
```diff
@@ -109,6 +109,7 @@ git clone https://github.com/OpenAccess-AI-Collective/axolotl
...
```
Comment (NanoCode012) [inline README.md]:
```suggestion
pip3 install packaging ninja
```

File: README.md (hunk 1 above)
Comment (melvinebenezer) [reply README.md]: agree

File: README.md (hunk 2 above)
Comment (melvinebenezer) [reply README.md]: agree

**Review by NanoCode012 — APPROVED** [review]

**Pull request merged**